src/openPr.js

```diff
@@ -150,12 +171,14 @@ module.exports = async function ({ context, inputs, packageVersion }) {
   } catch (err) {
     let message = `Unable to create the pull request ${err.message}`
     try {
-      await execWithOutput('git', ['push', 'origin', '--delete', branchName])
+      const exec = execWithOutput()
+      await exec('git', ['push', 'origin', '--delete', branchName])
```

Comment:
it's not really for readability but to avoid having to pass the cwd parameter for all function calls. Since most of the time when using this function all calls will be related and run on the same directory it makes sense for me to pass it once and then it will be set for all subsequent calls.